tests/pools/test_pool_rpc.py

```diff
@@ -372,6 +374,66 @@ async def test_create_multiple_pool_wallets(self, one_wallet_node_and_rpc, fee,
         with pytest.raises(ValueError):
             await client.pw_status(3)
 
+        def wallet_is_synced():
+            return (
+                wallet_node_0.wallet_state_manager.blockchain.get_peak_height()
+                == full_node_api.full_node.blockchain.get_peak_height()
+            )
+
+        # Create some CAT wallets to increase wallet IDs
+        for i in range(5):
+            res = await client.create_new_cat_and_wallet(20)
+            await asyncio.sleep(2)
+            summaries_response = await client.get_wallets()
+            assert res["success"]
+            cat_0_id = res["wallet_id"]
+            asset_id = bytes.fromhex(res["asset_id"])
+            assert len(asset_id) > 0
+            await self.farm_blocks(full_node_api, our_ph_2, 6)
+            await time_out_assert(20, wallet_is_synced)
+            bal_0 = await client.get_wallet_balance(cat_0_id)
+            assert bal_0["confirmed_wallet_balance"] == 20
+
+        # Test creation of many pool wallets. Use untrusted since that is the more complicated protocol, but don't
+        # run this code more than once, since it's slow.
+        if fee == 0 and not trusted:
+            for i in range(22):
+                creation_tx_3: TransactionRecord = await client.create_new_pool_wallet(
+                    our_ph_1, "localhost", 5, "localhost:5000", "new", "FARMING_TO_POOL", fee
+                )
+                await time_out_assert(
+                    10,
+                    full_node_api.full_node.mempool_manager.get_spendbundle,
+                    creation_tx_3.spend_bundle,
+                    creation_tx_3.name,
+                )
+                await self.farm_blocks(full_node_api, our_ph_2, 2)
+                await time_out_assert(20, wallet_is_synced)
+
+                full_config: Dict = load_config(wallet_0.wallet_state_manager.root_path, "config.yaml")
+                pool_list: List[Dict] = full_config["pool"]["pool_list"]
+                assert len(pool_list) == i + 3
+                if i == 0:
+                    # Ensures that the CAT creation does not cause pool wallet IDs to increment
+                    summaries_response = await client.get_wallets()
+                    print(summaries_response)
```

Comment:
```suggestion
```

Reply:
Im not sure if you want to keep all those `print` calls in the whole test. Looked to me like debugging thats why i proposed to drop them all.

And in this and the below case also the `client.get_wallets` seems redundant?

Reply:
good point thanks